python/libraft/pyproject.toml

```diff
@@ -45,6 +45,41 @@ known-first-party = [
 
 [tool.ruff.lint.isort.sections]
 
+[tool.isort]
+line_length = 79
+multi_line_output = 3
+include_trailing_comma = true
+force_grid_wrap = 0
+combine_as_imports = true
+order_by_type = true
+known_first_party = [
+    "libraft",
+]
+default_section = "THIRDPARTY"
+sections = [
+    "FUTURE",
+    "STDLIB",
+    "THIRDPARTY",
+    "DASK",
+    "RAPIDS",
+    "FIRSTPARTY",
+    "LOCALFOLDER",
+]
+skip = [
+    "thirdparty",
+    ".eggs",
+    ".git",
+    ".hg",
+    ".mypy_cache",
+    ".tox",
+    ".venv",
+    "_build",
+    "buck-out",
```

Comment:
Some of this stuff might be extremely dated, so copy-paste might not be ideal. I don't know if `buck-out` is relevant. Similarly, we don't use Mercurial (`.hg`). `.eggs` and `.tox` are probably not needed either.